### PR TITLE
v3.0

### LINKED ISSUE #70: DictionaryBundle v3.0
- [x] Remove deprecation
- [x] Full SF4 dependency injecton definition
- [x] Dictionary merge and inheritance (#69)
- [x] Dictionary implements Countable

### PR DESCRIPTION
**Pull request by PedroTroller**
Closes #70

### CODE REVIEW

**Comment by pbowyer** [comment]
Is there anything we can do to help get `v3.0` released?

**Comment by PedroTroller** [comment]
@pbowyer The v2.3 is finished I think. So as soon as it will be merged, le 3.0 will be rebased and also merged.

**Comment by PedroTroller** [comment]
@pbowyer the v3.0 is ready for review. If you want, you can review le code changes and I'm waiting for @AntoineLelaisant review to merge en release the version.

File: src/Knp/DictionaryBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -19,37 +19,38 @@ public function getConfigTreeBuilder()
...
-                ->useAttributeAsKey('name')
-                    ->prototype('array')
-                        ->beforeNormalization()
-                            ->always()
-                            ->then(function ($values) {
-                                if (false === array_key_exists('type', $values)) {
-                                    if (false === array_key_exists('content', $values)) {
-                                        return ['type' => 'value', 'content' => $values];
-                                    }
+            ->useAttributeAsKey('name')
```
Comment (AntoineLelaisant) [inline src/Knp/DictionaryBundle/DependencyInjection/Configuration.php]: I know you're kinda strict about cs fixing, but for this specific file, the indentation helps a lot to make things more readable. So I think we should ignore this class from cs fixing :)

**Pull request merged**